### PR TITLE
Bug 1694542 - Hide View CSV action when no installed CSV

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1694542

/cc @alecmerdler @rhamilto

### CODE REVIEW

File: frontend/public/components/operator-lifecycle-manager/subscription.tsx [hunk 1]
```diff
@@ -38,10 +38,14 @@ const menuActions = [
...
-  (kind, obj) => ({
-    label: `View ${ClusterServiceVersionModel.kind}...`,
-    href: `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${_.get(obj.status, 'installedCSV')}`,
-  }),
+  (kind, obj) => {
+    const installedCSV = _.get(obj, 'status.installedCSV');
+    return {
+      label: `View ${ClusterServiceVersionModel.kind}...`,
+      href: `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${installedCSV}`,
+      hidden: !installedCSV,
```
Comment (alecmerdler) [inline frontend/public/components/operator-lifecycle-manager/subscription.tsx]: Nice, I didn't realize we had a `hidden` argument for kebab actions.

**Comment by spadgett** [comment]
Ran the Operator Hub tests locally without failure. They're currently broken in CI :(

```
Summary:

Suites:  6 of 6
Specs:   51 of 53 (2 pending)
Expects: 63 (0 failures)
Finished in 135.376 seconds
```

/skip

**Pull request merged**